modal/cli/launch.py

```diff
@@ -18,9 +18,7 @@
     name="launch",
     no_args_is_help=True,
     help="""
-    [Preview] Open a serverless app instance on Modal.
-
-    This command is in preview and may change in the future.
+    Open a serverless app instance on Modal.
```

Comment:
I think we still consider these interfaces fairly unstable fwiw (but can remove the `[Preview]` from the top-level description.